cirno/Vector.cs

```diff
@@ -32,6 +32,11 @@ public struct Vector {
                 && Math.Abs(Y - other.Y) < tolerance;
         }
 
+        public float Distance(Vector point)
+        {
+            return (float) Math.Sqrt(Math.Pow(X - point.X, 2) + Math.Pow(Y - point.Y, 2));

```

Comment:
Just one more formatting issue, 
```csharp
return (float) Math.Sqrt
```
Can you remove whitespace after type casts?

Reply:
Done

Reply:
Thanks for fast response, btw, before you commend I already merged lol